jscomp/build_tests/cli_help/input.js

```diff
@@ -72,194 +72,204 @@ const dumpHelp =
   "Usage: rescript dump <options> [target]\n" +
   "`rescript dump` dumps the information for the target\n";
 
-// Shows build help with --help arg
-let out = child_process.spawnSync(`../../../rescript`, ["build", "--help"], {
-  encoding: "utf8",
-  cwd: __dirname,
-});
-assert.equal(out.stdout, buildHelp);
-assert.equal(out.stderr, "");
-assert.equal(out.status, 0);
+async function test() {
+  {
+    // Shows build help with --help arg
+    const out = await exec(`../../../rescript`, ["build", "--help"], {
```

Comment:
If we are already refactoring this, we could add a helper function in this file so that we can just write

```js
const out = await execReScript(["build", "--help"]);
```

Reply:
Can I separate it into a separate pure refactor PR?

Aside from the cli_help test, there are many other places to change.

Reply:
Yes, of course!